envoy/http/codec.h

```diff
@@ -165,6 +166,12 @@ class ResponseEncoder : public virtual StreamEncoder {
    * error.
    */
   virtual bool streamErrorOnInvalidHttpMessage() const PURE;
+
+  /**
+   * Set a new request decoder for this ResponseEncoder.
+   * @param decoder new request decoder.
```

Comment:
think it's worth a comment on why this is a useful association?

Reply:
added, ptal!